PythonAPI/contrib/CppApi/coco.h

```diff
@@ -0,0 +1,135 @@
+// Copyright 2019 The TensorFlow Authors. All Rights Reserved.
```

Comment:
How about move the files from PythonAPI/contrib/CppApi/coco.h to contrib/CppApi/coco.h, make contrib and PythonAPI at the same level?

Reply:
done.